### PR TITLE
[route_check]: Ignore standalone tunnel routes

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <lawlee@microsoft.com>

#### What I did
When checking for route entry mismatches, ignore mismatched routes where an APPL_DB neighbor entry with a zero MAC is present. 

These routes are introduced by this change in SWSS and are expected: https://github.com/sonic-net/sonic-swss/pull/2137

#### How to verify it
Run the updated `route_check_test.sh` script

### CODE REVIEW

File: scripts/route_check.py [hunk 1]
```diff
@@ -450,6 +450,42 @@ def filter_out_vnet_routes(routes):
...
+    neigh_keys = neigh_table.getKeys()
+    standalone_tunnel_route_ips = []
+    updated_routes = []
+
+    for neigh in neigh_keys:
+        _, mac = neigh_table.hget(neigh, 'neigh')
+        if mac == '00:00:00:00:00:00':
+            # remove preceding 'VlanXXXX' to get just the neighbor IP
+            neigh_ip = ':'.join(neigh.split(':')[1:])
+            standalone_tunnel_route_ips.append(neigh_ip)
```
Comment (prsunny) [inline scripts/route_check.py:473]: Please return if the `standalone_tunnel_route_ips` is empty

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
You may want to add few more test cases and also please raise a separate PR to 202012

**Review by prsunny — DISMISSED** [review]

**Review by prsunny — APPROVED** [review]

**Comment by yxieca** [comment]
@theasianpianist is there another PR needs to be cherry-picked before this one? There is a conflict. Can you raise separate PR?

**Comment by theasianpianist** [comment]
> @theasianpianist is there another PR needs to be cherry-picked before this one? There is a conflict. Can you raise separate PR?

PR here https://github.com/sonic-net/sonic-utilities/pull/2346

**Pull request merged**